client/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -149,6 +202,71 @@ describe('FeedbackGeneratorService', function() {
     });
   });
 
+ /*
+  TODO: Having a hard time getting coverage for getFeedback else branch
```

Comment:
TODOs should have a username for the person responsible for figuring out and fixing the TODO (usually the person who wrote the TODO). From reading this comment, I'm unsure as to what you're trying to do, so please be more clear with what you mean by getFeedback else branch.